site/content/en/contributions/RELEASING.md

```diff
@@ -150,15 +150,24 @@ export GITHUB_REMOTE=origin
       {{- "v1.2.0" -}}
       {{- end -}}
       ```
-      
+
+   1. Update `site/hugo.toml`, add the new version to the `params.versions` section.
+
+      ```console
+      [[params.versions]]
+        version = "v1.3"
+        url = "/v1.3"
+        eol = "2025-07-30"
+      ```
+
    1. Update `site/hugo.toml`, change the version to current major version.
```

Comment:
number 2 ?

Reply:
Markdown format allows always using 1 for order lists. The render will convert it to the correct number.

Reply:
got it thanks